modules/dmrpp_module/DmrppArray.cc

```diff
@@ -946,27 +946,27 @@ void DmrppArray::read_chunks_unconstrained()
     super_chunks.push(current_super_chunk);
 
     // Make the SuperChunks using all the chunks.
-    for(const auto& chunk: get_immutable_chunks()){
+    for(const auto& chunk: get_immutable_chunks()) {
         bool added = current_super_chunk->add_chunk(chunk);
-        if(!added){
+        if (!added) {
```

Comment:
Is this a style thing you want us all to adopt?

Reply:
Yes. It's also the default for lots of automated tools.